spec/deck_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rspec'
+require './lib/card'
+require './lib/deck'
+
+RSpec.describe Deck do
+    before :each do
+        @card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+        @card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+        @card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+        @cards = [@card_1, @card_2, @card_3]
+        @deck = Deck.new(@cards) 
+    end
+
+    it 'class should exist' do
+        expect(@deck).to be_instance_of Deck
+    end
+
+    it 'should return cards' do
+        expect(@deck.cards).to eq(@cards)
+    end
+
+    it 'should return the count of card' do
+        expect(@deck.count).to eq(3)
+    end
+
+    it 'should return the category' do
+        expect(@deck.cards_in_category(:STEM)).to eq([@card_2, @card_3])
+    end
+
```

Comment:
This test looks great! The only thing I'll add, is that it might be nice to add more assertions to the last test where you're testing `cards_in_category`. For example, maybe you make another assertion like: 
`expect(@deck.cards_in_category(:Geography)).to eq([@card_1])`

This will help make sure that your code is dynamic and working for any category that gets passed in to the method